server.js

```diff
@@ -53,26 +66,44 @@ app.post('/login_or_create_user', function (req, res) {
 
 // This is the endpoint the link in the magic link hits. It takes the token from the
 // link's query params and hits the stytch authenticate endpoint to verify the token is valid
-app.get('/authenticate', function (req, res) {
+app.get('/authenticate', async (req, res) => {
     const queryObject = url.parse(req.url,true).query;
-    client.magicLinks.authenticate(queryObject.token)
-        .then(
-            // on success render the logged in view
-            res.render('loggedIn')
-        )
-        .catch(err => {
-            // on failure, log the error then render the homepage
-            console.log(err)
-            res.render('loginOrSignUp')
-        });
+
+    const resp = await client.magicLinks.authenticate({
+        session_duration_minutes: 60,
+        token: queryObject.token,
```

Comment:
Same as my other comment RE: is this an older SDK with a different syntax. The latest version of the SDK has a property `magic_links_token` instead of `token`. Also it has a different request for Discovery Magic Links (`client.magicLinks.discovery.authenticate`).

Reply:
oooh i think you're looking at the B2B client -- this is the B2C one!

Reply:
You are SO right, resolving this comment.